jina/executors/encoders/nlp/transformer.py

```diff
@@ -2,76 +2,92 @@
 __license__ = "Apache-2.0"
 
 import os
-
-import numpy as np
-
+from typing import Optional
```

Comment:
numpy as np needed for type hint, lint is complaining about it

Reply:
I think it's a bit hacky way, maybe explicitly use `noqa` statement if linter is an only issue? Or just remove np from type hint?

Reply:
On the second thought, why not, let's return it back. Generally, there are a lot of place when `if False: import ...` are used. My comment was about this situation.